test/ruleLoaderTests.ts

```diff
@@ -17,8 +17,7 @@
 import {loadRules} from "./lint";
 
 describe("Rule Loader", () => {
-    const path = require("path");
-    const rulesDirectory = path.join(global.process.cwd(), "build/rules");
+    const rulesDirectory = "build/src/rules";
```

Comment:
The old code actually didn't even work, but we never noticed since TSLint just ignored bad paths to rule directories. A relative path is based off of the cwd by default, so no need to do that path.join